### PR TITLE
Issue78

### LINKED ISSUE #78: Separate hello world code examples from source, move to external file
**Who are you?**
Developer

**What feature do you want?**
All hello world code examples to be moved to an external file, separate from bot code

**Why do you want this feature?**
Code clutter and ease of management

### PR DESCRIPTION
**Pull request by tux2603**
# Description

Modified the `helloworld` command to read language data from a yaml file instead of a hard coded python dict.

## Issues

Closes #78 

## Type of change

Select one or more of the following:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other (describe below)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. List any edge cases you tested.
If you come up with any edge cases you didn't test while writing this PR, cancel the PR and test again.

- [x] Ran `-helloworld ls`
- [x] Ran `-helloworld` ten times
- [x] Ran `-helloworld python`
- [x] Ran `-helloworld fortran`
- [x] Ran `-helloworld nope`

# Checklist:

- [x] All local commits have been pushed to remote
- [x] All changes on the base branch have been merged into this branch, either by rebase or merge
- [x] My code is [PEP-8](https://pep8.org/) compliant (excluding maximum line length, keep that to 100ish characters)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added [Google Docstrings](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) for all new functions/methods
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: Cogs/StudentCommands.py [hunk 1]
```diff
@@ -58,56 +59,44 @@ async def helloworld(self, ctx, language='random'):
...
-            languages = [i for i in outputs]
+        if language.lower() == 'ls':
+            languages = [i for i in language_data]
             languages.sort()
             languages = '\n'.join(languages)
             await ctx.send(f'I know:\n{languages}')
             return
 
         # If invalid input, make it random
         language = language.lower()
```
Comment (mbozada) [inline Cogs/StudentCommands.py:88]: I think this line should be earlier, since a .lower() was added to the comparison on line 80. Maybe put a comment of "Cleaned Input" followed by language = language.lower()? Up to you, this is a super nitpick.

**Review by mbozada — APPROVED** [review]
Everything looks good. I had one small nitpick, but the change looks very clean.

**Review by CodingPenguin1 — APPROVED** [review]

**Comment by tux2603** [comment]
Ryan we weren't done lol
